config.md

```diff
@@ -56,14 +56,14 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
   This value MUST be an absolute path.
   * Windows: one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
   * Solaris: corresponds to "dir" of the fs resource in [zonecfg(1M)][zonecfg.1m].
-* **`type`** (string, OPTIONAL) The filesystem type of the filesystem to be mounted.
-  * Linux: valid *filesystemtype* supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
+* **`type`** (string, OPTIONAL) The type of the filesystem to be mounted.
+  * Linux: the types of filesystem supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
```

Comment:
This still reads awkwardly to me, since there are multiple filesystems listed in `/proc/filesystems`.  Maybe we can drop the “type” echo:

> * Linux: filesystems supported by the kernel are listed in `/proc/filesystems` (e.g., `minix`, `ext2`, `ext3`, …).